### ISSUE TITLE
loss will increase much large, only in some step

### ISSUE THREAD

**Issue by hdmjdp** [NVIDIA/waveglow#32]
step_78540:	loss_-4.065555573
step_78560:	loss_-4.490265846
step_78580:	loss_-4.454271317
step_78600:	loss_-4.322495461
step_78620:	loss_4191.683593750
step_78640:	loss_-2.408614397
step_78660:	loss_-2.908127546

**Comment by rafaelvalle** [comment]
Please share you config.json file.

**Comment by maozhiqiang** [comment]
the loss  explode

**Comment by hdmjdp** [comment]
@rafaelvalle 
{
    "train_config": {
        "output_directory": "checkpoints",
        "epochs": 100000,
        "learning_rate": 1e-4,
        "sigma": 1.0,
        "iters_per_checkpoint": 2000,
        "batch_size": 1,
        "seed": 1234,
        "checkpoint_path": ""
    },
    "data_config": {
        "wav_dataroot":"/home/hdm/Documents/tts/data/wav/hb_cen_lily_sent-24K",
        "mel_dataroot":"/home/hdm/Documents/wavenet/msc",
        "segment_length": 18000,
        "sampling_rate": 24000,
        "filter_length": 2048,
        "hop_length": 120,
        "win_length": 2048,
        "mel_fmin": 0.0,
        "mel_fmax": 8000.0
    },
    "dist_config": {
        "dist_backend": "nccl",
        "dist_url": "tcp://localhost:54321"
    },

    "waveglow_config": {
        "n_mel_channels": 80,
        "n_flows": 12,
        "n_group": 8,
        "n_early_every": 4,
        "n_early_size": 2,
        "WN_config": {
            "n_layers": 8,
            "n_channels": 512,
            "kernel_size": 3
        }
    }
}

**Comment by rafaelvalle** [comment]
The loss could have such a spike if your dataset has an audio sample that is all "silence".

**Comment by rafaelvalle** [comment]
@maozhiqiang let's address https://github.com/NVIDIA/waveglow/issues/38 here as well.

**Comment by hdmjdp** [comment]
[图片]

**Comment by jiqizaisikao** [comment]
Hi,the max p=1 and min(-log(p))=0,so why the loss <0

**Comment by rafaelvalle** [comment]
I mean there can be an audio slice that has only silence.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
